presto-oracle/src/main/java/io/prestosql/plugin/oracle/OracleClient.java

```diff
@@ -126,8 +127,6 @@
 
     private static final int PRECISION_OF_UNSPECIFIED_NUMBER = 127;
 
-    private static final int ORACLE_MAX_LIST_EXPRESSIONS = 1000;
```

Comment:
With this you are changing the default value to 32. Why so?

Reply:
It provides best balance in general use. Large IN lists really cause significant perf regresions.